src/index.ts

```diff
@@ -3,8 +3,17 @@ type LocalitConfig = {
   type?: "localStorage" | "sessionStorage";
 };
 
+type LocalitStore = {
+  value:any,
+  meta: LocalitMetadata
+}
+
+type LocalitMetadata = {
+  created: number,
+  expire?: number
```

Comment:
Perfect .. yes for me is easier to read createdAt and expiresAt.
At the moment `created` is not used, but I think it would be a good property to use in the future.
I remove the prop for the moment.